README.md

```diff
@@ -12,9 +15,9 @@ HealthyHome is an open-source smart home device which monitors the indoors envir
 
 The project is made up of several software components that are placed in the repository in the following structure:
 
-- `monitoring-server/`: the monitoring server that collects and processes all the data from the server
-- `logging-server/`: the data logging server which keeps track of the realtime data using a document store and builds statistics
-- `mobile-app/`: a mobile application for phones which interfaces the users with the smart device
+-   `monitoring-server/`: the monitoring server that collects and processes all the data from the server
+-   `logging-server/`: the data logging server which keeps track of the realtime data using a document store and builds statistics
+-   `mobile-app/`: a mobile application for phones which interfaces the users with the smart device
```

Comment:
How come the extra space? Does not seem to affect the final rendering at all

Reply:
oh interesting I guess it's doing a full 4 spaces (including the - in this case) tab as a regular indentation, makes sense!